src/murmur/Server.cpp

```diff
@@ -849,7 +849,7 @@ void Server::run() {
 					case MessageHandler::UDPVoiceCELTAlpha:
 					case MessageHandler::UDPVoiceCELTBeta:
 						if (bOpus)
-							break;
+							break; // Intentional fallthrough
```

Comment:
How about this instead?

	if (msgType == MessageHandler::UDPVoiceSpeex ||
	    msgType == MessageHandler::UDPVoiceCELTAlpha ||
	    msgType == MessageHandler::UDPVoiceCELTBeta ||
	    msgType == MessageHandler::UDPVoiceOpus) {

	    // Determine if the packet should be processed
	    // by Murmur. Clients can still send non-Opus
	    // voice packets when in Opus mode. However,
	    // Murmur should discard them in that case.
	    bool ok = true;
	    if (bOpus && msgType != MessageHandler::UDPVoiceOpus) {
	   		ok = false;
	    }

	    if (ok) {
	        u->aiUdpFlag = 1;
	        processMsg(u, buffer, len);
	    }
	} else if (msgType == MessageHandler::UDPPing) {
	    QByteArray qba;
	    sendMessage(u, buffer, len, qba, true);
	}

Reply:
It looks much better and simpler to understand, but I would use a direct statement instead of creating a variable for it.